src/database/queries/bulkUploads/selectWithPagination.sql

```diff
@@ -1,10 +1,10 @@
-SELECT
-  id as "id",
-  file_name as "fileName",
-  source_key as "sourceKey",
-  status as "status",
-  file_size as "fileSize",
-  created_at AS "createdAt"
+SELECT bulk_upload_to_json(bulk_uploads.*) as "object"
 FROM bulk_uploads
 ORDER BY id DESC
-OFFSET :offset FETCH NEXT :limit ROWS ONLY
+LIMIT
+  CASE WHEN :limit != 0 THEN
+    :limit
+  ELSE
+    NULL
+  END
+OFFSET :offset
```

Comment:
Why the switch from OFFSET/FETCH NEXT to LIMIT/OFFSET? It looks like this works, for example: `select name, setting, unit from pg_settings offset null fetch next null rows only;`

Reply:
Oh huh, I thought `fetch next null rows only;` would only return *one* row, and I couldn't get `ALL` to work without it yelling at me.

Was I misreading the docs 😅

Reply:
I'm gonna merge this as is, but when I continue this refactor I'll actually try out the `fetch null` thing to see if it returns all rows or just 1 row; I'll call out what I learn in that next PR!